HomePage.html

```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
 						</ul>
 
 					<button class="scroll-button">
-						<img src="Assets/Icons/scroll-button.svg" alt=""/>
+						<a href="#projects"><img src="Assets/Icons/scroll-button.svg" alt=""/></a>
 					</button>
 		
 				</section>
 	
+		</div>
+
+		
+		<div class="project-page" id="projects">
+			
+			<section >
+				<h1>My Recent <br> Works</h1>
+				<div class="underline"></div>
+			</section>
+		
+		
+			<div class="grid-card-display">
+				
+				<!-- a card element contains the image in project-card-image and details in project-card-details -->
+				<section class="project-card">
+					<div class="project-card-image">
+								<img src="Assets/Images/gym-fit.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Html</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
+					
+						</div>
+			
+				</section>
+				
+				<section class="project-card">
+					<div class="project-card-image">
+								<img src="Assets/Images/art-printing.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Typescript</li>
+										<li>Html</li>
+										<li>Java</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
```

Comment:
[REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/192018711-8fce6d47-a744-49e0-8755-b2634fb467b5.png)|

Kindly follow the colour pattern for all your buttons. Just the green ones

|Template|
|---------|
|![image](https://user-images.githubusercontent.com/62938735/192021460-0ae86b33-7c4d-4b84-a698-67d5d608c897.png)|